aequilibrae/project/network/network.py

```diff
@@ -276,7 +276,7 @@ def build_graphs(self) -> None:
             warn(f'Fields were removed form Graph for being non-numeric: {",".join(removed_fields)}')
 
         curr.execute('select node_id from nodes where is_centroid=1;')
-        centroids = np.array([i[0] for i in curr.fetchall()])
+        centroids = np.array([i[0] for i in curr.fetchall()], np.int32)
```

Comment:
type here is different from graph.py, is that on purpose?

Reply:
Not on purpose. Good catch